src/Cameras/Inputs/freeCameraDeviceOrientationInput.ts

```diff
@@ -42,6 +51,10 @@ export class FreeCameraDeviceOrientationInput implements ICameraInput<FreeCamera
     private _beta: number = 0;
     private _gamma: number = 0;
 
+    /**
+     * @hidden
+     */
+    public _onDeviceOrientationChangedObservable = new Observable<void>();

```

Comment:
needs to be clear at dispose time

Reply:
This class is not disposable so I added a clear for this when the attached camera is disposed.